### PR TITLE
Campaign/applytheme sidebar header bg

### PR DESCRIPTION
**Pull request by deanwhillier**
#### Summary
ApplyTheme Campaign: This PR completes the merge of the theme.sidebarHeaderBg commits from the campaign branch.

Dev approval for the changes was completed on the original PR(s), so we are just looking for rubber stamp approvals by dev's to satisfy merge requirements.

As QA was skipped on PR merges into campaign branches, this PR will need proper QA before merging. See below for details on what to QA for this PR.

#### Ticket Link

* https://github.com/mattermost/mattermost-server/issues/15867
* https://github.com/mattermost/mattermost-server/issues/15868
* https://github.com/mattermost/mattermost-server/issues/15869
* https://github.com/mattermost/mattermost-server/issues/15870

#### Related Pull Requests
See above tickets for related PRs

### CODE REVIEW

File: sass/layout/_webhooks.scss [hunk 1]
```diff
@@ -352,4 +353,4 @@
             }
         }
     }
-}
+}
```
Comment (deanwhillier) [inline sass/layout/_webhooks.scss:356]:
```suggestion
}

```

File: sass/components/_emoticons.scss [hunk 2]
```diff
@@ -81,6 +81,10 @@
     width: 315px;
     z-index: 999;
 
+    .emoji-picker__header{
+        background: var(--sidebar-header-bg);
+    }
```
Comment (deanwhillier) [inline sass/components/_emoticons.scss:86]: @jgilliam17, this applies the background colour of the emoji picker header in mobile view.

File: sass/layout/_webhooks.scss [hunk 3]
```diff
@@ -108,6 +108,7 @@
             border-left-style: solid;
             border-left-width: 4px;
             padding: 12px;
+            border-left-color: var(--sidebar-header-bg);
```
Comment (deanwhillier) [inline sass/layout/_webhooks.scss:111]: This affects left border color of the detail boxes bots like Jira post:

Screen Shot 2021-01-26 at 1 20 31 PM

File: sass/responsive/_mobile.scss [hunk 4]
```diff
@@ -1369,7 +1369,7 @@
 
     .search-bar__container {
         @include flex(0 0 50px);
-        background: $primary-color;
+        background: var(--sidebar-header-bg);
```
Comment (deanwhillier) [inline sass/responsive/_mobile.scss:1372]: @jgilliam17, this affects the background colour for the search bar header in mobile.

**Review by jgilliam17 — APPROVED** [review]
Thank you @deanwhillier 
Tested, looks good to merge.
- Verified emoji picker header and the search bar header background color in mobile view. Also, verified background color of various modal headers and left border color of the detail boxes - all as expected.

**Review by hmhealey — APPROVED** [review]

**Review by nevyangelova — APPROVED** [review]
LGTM

**Pull request merged**